src/nvim/drawline.c

```diff
@@ -2265,7 +2265,7 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, int col_rows, b
         }
 
         if (wp->w_buffer->terminal) {
-          wlv.char_attr = hl_combine_attr(term_attrs[wlv.vcol], wlv.char_attr);
+          wlv.char_attr = hl_combine_attr(TERM_ATTRS_MAX > wlv.vcol ? term_attrs[wlv.vcol] : 0, wlv.char_attr);
```

Comment:
This is a bit hard to read
```suggestion
          wlv.char_attr = hl_combine_attr(wlv.vcol < TERM_ATTRS_MAX ? term_attrs[wlv.vcol] : 0, wlv.char_attr);
```

Reply:
Thats fair, i'll fix that